examples/lua_REPL/Makefile

```diff
@@ -6,11 +6,12 @@ BOARD ?= native
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
-BOARD_BLACKLIST := arduino-duemilanove arduino-leonardo \
-                   arduino-mega2560 arduino-nano arduino-uno atmega328p \
-                   chronos hifive1 hifive1b mega-xplained microduino-corerf \
-                   msb-430 msb-430h pic32-clicker pic32-wifire telosb \
-                   waspmote-pro wsn430-v1_3b wsn430-v1_4 z1
+BOARD_BLACKLIST +=  \
+    hifive1         \
+    hifive1b        \
+    pic32-wifire    \
+    pic32-clicker   \
```

Comment:
Here and below: `FEATURES_BLACKLIST += arch_mips32r2 arch_riscv` to `pkg/lua`?

Reply:
Done for MIPS. For RISCV, I don't see any reason why the boards is blacklisted. Will see if I can find some info in related PRs.

Reply:
For RISCV, the problem is missing functions in the newlib nano provided by the toolchain. Not sure if simply blacklisting them is the way to go but I did it anyway.

Reply:
What alternative would you propose?